2001-3000/LIT2222RetuaH.xml

```diff
@@ -42,8 +59,114 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
-      <body>
-         <ab/>
-      </body>
+  <body>
+     <div type="bibliography">
+        <listBibl type="secondary">
+           <bibl>
+              <ptr target="bm:Getachew2010RetuaHaymanot"/>
+           </bibl>
+           <bibl>
+             <ptr target="bm:Villa2019Filologia"/>
+             <citedRange unit="page">147-154</citedRange>
+            </bibl>
+            <bibl>
+            <ptr target="bm:Getatchew1981Controversies"/>
+            <citedRange unit="page">109-116</citedRange>
+            </bibl>
+         </listBibl>
+         </div>
+
+<div type="edition">
+  <note>Following text structure is based on <ref type="mss" corresp="EMML2375">EMML 2375</ref>.</note>
+
+<div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Introduction">
+<div type="textpart" subtype="incipit">
+   <ab>ስሉሥ፡ ቅዱስ፡ ዘያሌቡ፡ ወኢይትሌቡ፡ ዘያዔውቅ፡ ወኢይትዓወቅ፡ ዘያኤምር፡ ወኢይትአመር፡ ዘይሄሊ፡ ወኢይትሀለይ፡ ዘይጤይቅ፡ ወኢይጠየቅ፡ ዘይእኅዝ፡ ወኢይትአኃዝ፡</ab>
+</div>
+</div>
+
+<div type="textpart" subtype="homily" xml:lang="gez" xml:id="Nativity" corresp="LIT2113OntheB">
+<label>On Christ's nativity</label>
+</div>
+
+<div type="textpart" subtype="homily" xml:lang="gez" xml:id="Baptism" corresp="LIT2095OnHowO">
+<label>On Christ's baptism</label>
+</div>
+
+<div type="textpart" subtype="homily" xml:lang="gez" xml:id="Beginning" corresp="LIT6060BeginningOfLent">
+<label>On fasting, prayer, and almsgiving, for the first Sunday of Lent</label>
+</div>
+
+<div type="textpart" subtype="homily" xml:lang="gez" xml:id="Sunday1" corresp="LIT6049SundayRetuaH">
+<label>On fasting and chastity, for the first Sunday of Lent</label>
+<note>Some manuscripts contain homilies only for the first three Sundays of Lent.</note>
+</div>
+
+<div type="textpart" subtype="homily" xml:lang="gez" xml:id="Sunday2" corresp="LIT6050SundayRetuaH">
+<label>On fasting and peace, for the second Sunday of Lent</label>
+</div>
+
+<div type="textpart" subtype="homily" xml:lang="gez" xml:id="Sunday3" corresp="LIT6061SundayRetuaH">
+<label>On fasting and meekness, for the third Sunday of Lent</label>
+</div>
+
+<div type="textpart" subtype="homily" xml:lang="gez" xml:id="Sunday4">
+</div>
```

Comment:
I think that labels here (even minimal, "For the fourth Sunday" or similar, would be helpful for readers.